app/models/reports/refreshable.rb

```diff
@@ -0,0 +1,24 @@
+module Reports
+  module Refreshable
```

Comment:
I wanted _one_ place where we call the Scenic method to refresh matviews, so consolidating that here....